code/modules/awaymissions/gateway.dm

```diff
@@ -246,6 +246,8 @@ var/obj/machinery/gateway/centerstation/the_gateway = null
 	if(istype(M, /mob/living/carbon))
 		if(exilecheck(M)) return
 	if(istype(M, /obj))
+		if(M.can_buckle && M.has_buckled_mobs())
+			if(exilecheck(M.buckled_mob)) return

```

Comment:
The return should be on a newline.

Reply:
Just following the style of the code around it, but sure, can do.

Reply:
Probably just old code, our contribution guide says that control statements should not have code on the same line as the statement so probably best to follow that to try and bring all code to the same standard :)

Reply:
Should I fix all instances of it in this file? Or just the one I added?

Reply:
That's up to you really.